index.html

```diff
@@ -22,10 +22,155 @@ <h1 class="profileName">Harshita Jain</h1>
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>JavaScript</li>
+                </ul>
+                <a href="#">
+                    <button type="button">
+                        <span>See this project</span>
+                        <img class="pr-button-img" src="images/Union.png">
+                    </button>
+                </a>
+            </div>
+        </div>
+        <div class="project">
+            <img class="pr-img" src="images/Rectangle 21 (4).png">
+            <div class="project-ins">
+                <h3>Project name goes here</h3>
+                <ul>
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>JavaScript</li>
+                </ul>
+                <a href="#">
+                    <button type="button">
+                        <span>See this project</span>
+                        <img class="pr-button-img" src="images/Union.png">
+                    </button>
+                </a>
+            </div>
+        </div>
+        <div class="project">
+            <img class="pr-img" src="images/Rectangle 21 (5).png">
+            <div class="project-ins">
+                <h3>Project name goes here</h3>
+                <ul>
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>JavaScript</li>
+                </ul>
+                <a href="#">
+                    <button type="button">
+                        <span>See this project</span>
+                        <img class="pr-button-img" src="images/Union.png">
+                    </button>
+                </a>
+            </div>
+        </div>
+       </div>
+    </section>
+    <section class="about">
+        <h2>About</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+        <button type="button">
+            Get my CV
+            <img src="images/ic_download.svg">
+        </button>
+        <div class="languages">
+            <h4 class="set-h">Languages</h4>
+            <ul class="set-ul">
+                <li>JavaScript(ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+            <img class="set-img" src="images/hash.png">
+        </div>
+        <div class="frameworks">
+            <h4 class="set-h">Frameworks</h4>
+            <ul class="set-ul">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <li>Selenium</li>
+            </ul>
+            <img class="set-img" src="images/Group 18.png">
+        </div>
+        <div class="skills">
+            <h4 class="set-h">Skills</h4>
```

Comment:
- [ Optional] Kindly align this h4 to the left to match the other sections and the Figma design.